### PR TITLE
ESLint and Prettier setup

### PR DESCRIPTION
**Pull request by khbrn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,11 +7,15 @@
 		"start:prod": "npx cross-env NODE_ENV=production node_modules/.bin/webpack-dev-server --mode production",
 		"start:dev": "npx cross-env NODE_ENV=development node_modules/.bin/webpack-dev-server --mode development",
 		"build:prod": "npx cross-env NODE_ENV=production node_modules/.bin/webpack  --mode production",
-		"build:dev": "npx cross-env NODE_ENV=development node_modules/.bin/webpack"
+		"build:dev": "npx cross-env NODE_ENV=development node_modules/.bin/webpack",
+		"lint": "eslint ./ --ignore-path .gitignore",
```
Comment (Vladlan) [inline package.json]: add .eslintignore

File: src/hooks/DummyHttp.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+import mockMovieData from '../mockData/mockMovieData';
+
+/*
+For now it works with mock data (without actual http request)
+*/
+export const useDummyHttp = (dependency) => {
```
Comment (Vladlan) [inline src/hooks/DummyHttp.js]: file name should be also be useDummyHttp

File: webpack.config.prod.js [hunk 3]
```diff
@@ -1,54 +1,54 @@
...
-			template: './src/index.html'
-		}),
-		new MiniCssExtractPlugin({
-			filename: '[name].bundle.css'
-		})
-	]
+  mode: "production",
+  entry: "./src/index.js",
+  output: {
+    filename: "[name].bundle.js",
```
Comment (Vladlan) [inline webpack.config.prod.js]: Add hash for fille names to invalidate cash in a browser when new version of the app is released https://webpack.js.org/guides/caching#output-filenames

File: src/components/App/App.js [hunk 4]
```diff
@@ -1,19 +1,66 @@
...
+
+  let fetchedMovieContent = (
+    <div className="loading-message__container ">
+      <h2 className="loading-message">Loading...</h2>
+    </div>
+  );
+
+  if (!isLoading) {
+    fetchedMovieContent = (
+      <>
```
Comment (Vladlan) [inline src/components/App/App.js]: 1. Just <Main/>
2. You can move loading indication inside Main

**Review by Vladlan — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (khbrn) [reply package.json]: done

File: src/hooks/DummyHttp.js (hunk 2 above)
Comment (khbrn) [reply src/hooks/DummyHttp.js]: done

File: src/components/App/App.js (hunk 4 above)
Comment (khbrn) [reply src/components/App/App.js]: done

**Pull request merged**